graph.lisp

```diff
@@ -80,7 +80,8 @@
 
 (defun open-graph (name location &key master-p slave-p master-host replication-port
                    replication-key package (buffer-pool-p t) (gc-heap-p t))
-  (let ((path (first (directory (ensure-directories-exist location))))
+  (ensure-directories-exist location)
```

Comment:
Did you check that this doesn't break sbcl and ccl?

Reply:
`directory` doesn't work in LispWorks, and I have tested this change in sbcl.